uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
         }
         return render('summary/food_eligible.xml', {'attendees': eligible})
 
+    def csv_import(self, message='', all_instances=None, form_link=None):
```

Comment:
That would probably be restrictive enough, though we haven't really been using that for awhile.  Though it would be kind of counterintuitive for "budget" access to be the thing used to grant import/export access.